src/transformState.js

```diff
@@ -4,8 +4,35 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type } = action;
```

Comment:
add here all properties instead of using destructuring assignment in the each case